src/Audit/Drupal/LargeDrupalFilesOfTypes.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drutiny\GovCMS\Audit\Drupal;
+
+use Drutiny\Audit;
+use Drutiny\Sandbox\Sandbox;
+use Drutiny\AuditResponse\AuditResponse;
+use Drutiny\Annotation\Param;
+use Drutiny\Annotation\Token;
+
+/**
+ * Identify files with a specified extensions which are larger than a specfied size. Pass if no matching files found.
+ * @Param(
+ *  name = "max_size",
+ *  description = "Report files larger than this value measured in bytes.",
+ *  type = "integer",
+ *  default = 10000000
+ * )
+ * @Param(
+ *  name = "extensions",
+ *  description = "File extensions to include.",
+ *  type = "array"
+ * )
+ * @Token(
+ *  name = "total",
+ *  description = "Total number of large files found",
+ *  type = "integer"
+ * )
+ * @Token(
+ *  name = "too_many_files",
+ *  description = "Text to display if there are more than 10 files found.",
+ *  type = "integer"
+ * )
+ * @Token(
+ *  name = "files",
+ *  description = "A list of up to 10 files that are too large.",
+ *  type = "integer"
+ * )
+ * @Token(
+ *  name = "plural",
+ *  description = "This variable will contain an 's' if there is more than one issue found.",
+ *  type = "string",
+ *  default = ""
+ * )
+ */
+class LargeDrupalFilesOfTypes extends Audit {
+
+  /**
+   * @inheritdoc
+   */
+  public function audit(Sandbox $sandbox) {
+    $max_size = (int) $sandbox->getParameter('max_size', 2000000);
+    $extensions = (array) $sandbox->getParameter('extensions', []);
+    $sandbox->setParameter('readable_max_size', $max_size / 1000 / 1000 . ' MB');
```

Comment:
It would be better to use a private function to translate the size to human readable format.

For example, 
$sandbox->setParameter('readable_max_size', $this->readableSize($max_size));